### ISSUE TITLE
from_dicts can produce incorrect CSV

### ISSUE THREAD

**Issue by lpil** [bcpeinhardt/gsv#10]
Hello!

When some of the fields are optional in the dicts `from_dicts` will output incorrect CSV.

Here's an example: https://github.com/gleam-lang/cookbook/pull/22/commits/02072d57457f597ad6dc131b151ba24032656204#diff-2ca70a8f4e13c5d17aa88262081cad9a6ae01d0d358553f872c42a12917d4432

Thanks,
Louis

**Comment by bcpeinhardt** [comment]
Yeah I was treating this as the first row being the row that specifies the headers, but I guess what needs to happen is to iterate over the entire List(Dict(String, String)) to produce the headers. I probably shouldn't have published so quickly I was pretty busy. I'll get a fix in asap.

**Comment by lpil** [comment]
Thank you 💜

**Comment by bcpeinhardt** [comment]
2.0.1 published.
